modules/tableEmbed.ts

```diff
@@ -52,36 +58,39 @@ export const composePosition = (delta: Delta, index: number) => {
   return newIndex;
 };
 
-const compactCellData = ({ content, attributes }) => {
+const compactCellData = ({ content, attributes }: CellData) => {
```

Comment:
We expect `content` to be a Delta.

Reply:
See comment here: https://github.com/quilljs/quill/pull/3845#discussion_r1287726794